content/blog/rag-pipelines-from-scratch/index.md

```diff
@@ -0,0 +1,423 @@
+---
+layout: blog-post
+title: "RAG Pipelines from scratch"
+description: "Let's build a simple RAG Pipeline with Haystack 2.0 by just connecting three components: a Retriever, a PromptBuilder and a Generator."
+featured_image: thumbnail.png
```

Comment:
You can add a `featured_image_caption` here if you want. That text will appear below the thumbnail